plugins/internal/tengoutil/secure_script.go

```diff
@@ -30,3 +36,66 @@ func NewSecureScript(input []byte, globals map[string]interface{}) (*tengo.Scrip
 
 	return s, nil
 }
+
+func createHTTPModule() map[string]tengo.Object {
+	return map[string]tengo.Object{
+		"get": &tengo.UserFunction{
+			Name: "get",
+			Value: func(args ...tengo.Object) (tengo.Object, error) {
```

Comment:
what if instead of defining `createHTTPModule`, we define the function in this line? and then put the `map[string]tengo.Object` into a global (or internal) variable.

Reply:
also, can we breakdown this function into smaller ones? for example, there are lines to get header:
```go
headers := make(map[string]string)
if len(args) == 2 {
...
```

we can move that into a dedicated function. and please check the subsequent lines.